### PR TITLE
Fix dropped errors

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: openpgp/packet/signature.go [hunk 1]
```diff
@@ -604,7 +604,7 @@ func (sig *Signature) Sign(h hash.Hash, priv *PrivateKey, config *Config) (err e
 func (sig *Signature) SignUserId(id string, pub *PublicKey, priv *PrivateKey, config *Config) error {
 	h, err := userIdSignatureHash(id, pub, sig.Hash)
 	if err != nil {
-		return nil
```
Comment (akalin-keybase) [inline openpgp/packet/signature.go:607]: Hmm, this seems to be fixed upstream: https://github.com/golang/crypto/blob/master/openpgp/packet/signature.go#L581 Time to rebase?

**Comment by maxtaco** [comment]
Yeah good idea.  It is usually quite painful though

**Comment by maxtaco** [comment]
LGTM

**Comment by akalin-keybase** [comment]
Briefly tried to rebase, but yeah, you're right, it seems painful. I'll leave it for another day. 😅

**Comment by maxtaco** [comment]
Probably a merge strategy is the way to go.

**Pull request merged**